[Issue] Debian pacakage generation for `rocm-profiler` failing

**Platform:** `Linux`
**Effected Archs:** `All the supported archs`

### Run:
- https://github.com/ROCm/TheRock/actions/runs/22347044236/job/64664224618
- https://github.com/ROCm/TheRock/actions/runs/22345488562/job/64658910857
- https://github.com/ROCm/TheRock/actions/runs/22344980762/job/64657152806
- https://github.com/ROCm/TheRock/actions/runs/22344854026/job/64656712038
- https://github.com/ROCm/TheRock/actions/runs/22344552571/job/64655680477


```
make[1]: Leaving directory '/home/runner/_work/TheRock/TheRock/output/packages/deb/amdrocm-profiler7.12.0~20260224'
   dh_installsystemduser -O--no-parallel
   dh_perl -O--no-parallel
   dh_link -O--no-parallel
   dh_strip_nondeterminism -O--no-parallel
   dh_compress -O--no-parallel
   dh_fixperms -O--no-parallel
   dh_missing -O--no-parallel
   dh_dwz -a -O--no-parallel
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/bin/rocprof-sys-avail: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/bin/rocprof-sys-causal: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/bin/rocprof-sys-instrument: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/bin/rocprof-sys-run: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/bin/rocprof-sys-sample: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/librocprof-sys-dl.so.1.5.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/librocprof-sys-rt.so.1.5.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/librocprof-sys-user.so.1.5.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/librocprof-sys.so.1.5.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-compute/librocprofiler-compute-tool.so: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libasm-0.188.so: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libcommon.so.13.0.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libdw-0.188.so: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libdynDwarf.so.13.0.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libdynElf.so.13.0.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libdyninstAPI.so.13.0.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libelf-0.188.so: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libgotcha.so.2.4.2: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libinstructionAPI.so.13.0.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libparseAPI.so.13.0.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libpatchAPI.so.13.0.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libpcontrol.so.13.0.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libstackwalk.so.13.0.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libsymtabAPI.so.13.0.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libtbb.so.2: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libtbbmalloc.so.2: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libtbbmalloc_proxy.so.2: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libunwind-coredump.so.0.0.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libunwind-ptrace.so.0.0.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libunwind-setjmp.so.0.0.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libunwind-x86_64.so.99.0.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libunwind.so.99.0.0: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-both-ndebug: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-both-rocprofsys-ndebug: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-cpu-ndebug: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-cpu-rocprofsys-ndebug: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-rng-ndebug: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-rng-rocprofsys-ndebug: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/hipMallocConcurrencyMproc: Unknown debugging section .debug_str_offsets
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/lib/libopenmp-target-lib.so: Unknown debugging section .debug_str_offsets
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/openmp-cg: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/openmp-lu: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/openmp-target: Unknown debugging section .debug_str_offsets
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/openmp-vv-host-test-parallel-for-simd-atomic: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/openmp-vv-host-test-team-default-shared: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/openmp-vv-offload-test-target-simd-if: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/openmp-vv-offload-test-target-teams-distribute-parallel-for-collapse: .debug_info section not present
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/roctx: Unknown debugging section .debug_str_offsets
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/transferBench: Unknown debugging section .debug_str_offsets
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/transpose: Unknown debugging section .debug_str_offsets
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/hipMallocConcurrencyMproc: Unknown debugging section .debug_str_offsets
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/lib/libopenmp-target-lib.so: Unknown debugging section .debug_str_offsets
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/openmp-target: Unknown debugging section .debug_str_offsets
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/roctx: Unknown debugging section .debug_str_offsets
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/transferBench: Unknown debugging section .debug_str_offsets
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/transpose: Unknown debugging section .debug_str_offsets
dwz: debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libpfm.so.4.13.0: DWARF compression not beneficial - old size 323605 new size 324677
dh_dwz: error: dwz -mdebian/amdrocm-profiler7.12/usr/lib/debug/.dwz/x86_64-linux-gnu/amdrocm-profiler7.12.debug -M/usr/lib/debug/.dwz/x86_64-linux-gnu/amdrocm-profiler7.12.debug -- debian/amdrocm-profiler7.12/opt/rocm/core-7.12/bin/rocprof-sys-avail debian/amdrocm-profiler7.12/opt/rocm/core-7.12/bin/rocprof-sys-causal debian/amdrocm-profiler7.12/opt/rocm/core-7.12/bin/rocprof-sys-instrument debian/amdrocm-profiler7.12/opt/rocm/core-7.12/bin/rocprof-sys-run debian/amdrocm-profiler7.12/opt/rocm/core-7.12/bin/rocprof-sys-sample debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/librocprof-sys-dl.so.1.5.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/librocprof-sys-rt.so.1.5.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/librocprof-sys-user.so.1.5.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/librocprof-sys.so.1.5.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-compute/librocprofiler-compute-tool.so debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libasm-0.188.so debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libcommon.so.13.0.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libdw-0.188.so debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libdynDwarf.so.13.0.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libdynElf.so.13.0.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libdyninstAPI.so.13.0.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libelf-0.188.so debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libgotcha.so.2.4.2 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libinstructionAPI.so.13.0.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libparseAPI.so.13.0.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libpatchAPI.so.13.0.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libpcontrol.so.13.0.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libpfm.so.4.13.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libstackwalk.so.13.0.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libsymtabAPI.so.13.0.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libtbb.so.2 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libtbbmalloc.so.2 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libtbbmalloc_proxy.so.2 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libunwind-coredump.so.0.0.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libunwind-ptrace.so.0.0.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libunwind-setjmp.so.0.0.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libunwind-x86_64.so.99.0.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/lib/rocprofiler-systems/libunwind.so.99.0.0 debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-both debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-both-ndebug debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-both-rocprofsys debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-both-rocprofsys-ndebug debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-cpu debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-cpu-ndebug debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-cpu-rocprofsys debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-cpu-rocprofsys-ndebug debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-rng debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-rng-ndebug debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-rng-rocprofsys debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/causal-rng-rocprofsys-ndebug debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/code-coverage debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/fork-example debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/hipMallocConcurrencyMproc debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/lib/libopenmp-target-lib.so debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/openmp-cg debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/openmp-lu debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/openmp-target debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/openmp-vv-host-test-parallel-for-simd-atomic debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/openmp-vv-host-test-team-default-shared debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/openmp-vv-offload-test-target-simd-if debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/openmp-vv-offload-test-target-teams-distribute-parallel-for-collapse debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/parallel-overhead debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/parallel-overhead-locks debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/rewrite-caller debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/roctx debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/thread-limit debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/trace-time-window debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/transferBench debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/transpose debian/amdrocm-profiler7.12/opt/rocm/core-7.12/share/rocprofiler-systems/examples/user-api returned exit code 1
dh_dwz: error: Aborting due to earlier error
make: *** [debian/rules:7: binary] Error 25
dpkg-buildpackage: error: fakeroot debian/rules binary subprocess returned exit status 2

In function: copy_package_contents
In function: copy_package_contents
Directory does not exist: /home/runner/_work/TheRock/TheRock/output/artifacts/rocprofiler-compute_dev_generic/profiler/rocprofiler-compute/stage
In function: copy_package_contents
In function: copy_package_contents
In function: copy_package_contents
In function: copy_package_contents
In function: copy_package_contents
In function: copy_package_contents
In function: generate_install_file
In function: package_with_dpkg_build
Error building deb packageamdrocm-profiler7.12.0~20260224: Command '['dpkg-buildpackage', '-uc', '-us', '-b']' returned non-zero exit status 2.

❌ Build aborted due to an error.

```

--- a/build_tools/packaging/linux/build_package.py
+++ b/build_tools/packaging/linux/build_package.py
@@ -372,7 +372,7 @@ def generate_debian_postscripts(pkg_info, deb_dir, config: PackageConfig):
     parts = config.rocm_version.split(".")
     if len(parts) < 3:
         raise ValueError(
-            f"Version string '{args.rocm_version}' does not have major.minor.patch versions"
+            f"Version string '{config.rocm_version}' does not have major.minor.patch versions"
         )
 
     env = Environment(loader=FileSystemLoader(str(SCRIPT_DIR)))
@@ -449,20 +449,16 @@ def package_with_dpkg_build(pkg_dir):
     Returns: None
     """
     print_function_name()
-    current_dir = Path.cwd()
-    os.chdir(Path(pkg_dir))
     # Build the command
     cmd = ["dpkg-buildpackage", "-uc", "-us", "-b"]
 
     # Execute the command
     try:
-        subprocess.run(cmd, check=True)
+        subprocess.run(cmd, check=True, cwd=pkg_dir)
         print(f"Deb Package built successfully: {os.path.basename(pkg_dir)}")
     except subprocess.CalledProcessError as e:
-        print(f"Error building deb package{os.path.basename(pkg_dir)}: {e}")
+        print(f"Error building deb package: {os.path.basename(pkg_dir)}: {e}")
         sys.exit(e.returncode)
-
-    os.chdir(current_dir)
 
 
 ######################## RPM package creation ####################

--- a/build_tools/packaging/linux/package.json
+++ b/build_tools/packaging/linux/package.json
@@ -999,6 +999,7 @@
       }
     ],
 
+    "Disable_DWZ": "True",
     "Gfxarch": "False"
   },
   {

